### PR TITLE
hunger game mission from MickaelFR

### PR DESCRIPTION
**Pull request by stephanegigandet**
French mission from MickaelFR (first draft, to be experimented)

### CODE REVIEW

File: lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<p>Tout à droite de l'écran (ou tout en bas de l'écran si vous êtes sur mobile), se trouvent des annotations qui vous permettent de savoir :</p>
+<ul>
+<li>Le nombre de produits restant à traiter (varie en fonction des filtres appliqués),</li>
+<li>Le nombre de produits annotés,</li>
+<li>Un historique des dernières actions menées sur votre session.</li>
+</ul>
+
+</div>  
+
+<p>Si vous avez des questions ou des remarques n'hésitez pas à nous contacter sur le Slack ou sur <a href="https://www.facebook.com/groups/374350705955208/permalink/902105433179730/">le groupe Facebook</a>.</p>
```
Comment (teolemon) [inline lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html:59]:
```suggestion
<p>Si vous avez des questions ou des remarques n'hésitez pas à nous contacter sur le Slack ou sur <a href="https://www.facebook.com/groups/openfoodfacts.fr/permalink/902105433179730/">le groupe Facebook</a>.</p>
```

Comment (stephanegigandet) [reply lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html:59]: Les 2 liens ne marchent pas en fait, c'est juste https://www.facebook.com/groups/openfoodfacts.fr

File: lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	<li><strong>category</strong> : portant sur le champs "catégorie" (Ex. Boissons, Bonbons, etc.)</li>
+	<li><strong>brand</strong> : portant sur la marque du produit (Ex. Sodébo, Charal, Alter Eco, etc.)</li>
+	<li><strong>product_weight</strong> : portant sur le poids du produit (Ex. 100 g, 1L, etc.)</li>
+</ul>
+<p>En dessous on trouve :</p>
+<ol>
+<li>Un champs de recherche, permettant de préciser sur quelle marque, un label, une catégorie on souhaite travailler,</li>
+<li>La question à laquelle il faut répondre en anglais. En français cela donne : "Ce produit correspond t-il à la valeur indiquée ?"</li>
+<li>L'image du produit,</li>
+<li>3 boutons clicables permettant de répondre à la question.</li>
```
Comment (teolemon) [inline lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html:35]:
```suggestion
<li>3 boutons cliquables permettant de répondre à la question.</li>
```

File: lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+<p>En haut de l'écran se trouvent 4 boutons (filtres) permettant de sélectionner le type de donnée que vous souhaitez compléter :</p>
+<ul>
+	<li><strong>label</strong> : portant sur le champs "label" (Ex. certification, origine du produit, etc.)</li>
+	<li><strong>category</strong> : portant sur le champs "catégorie" (Ex. Boissons, Bonbons, etc.)</li>
+	<li><strong>brand</strong> : portant sur la marque du produit (Ex. Sodébo, Charal, Alter Eco, etc.)</li>
+	<li><strong>product_weight</strong> : portant sur le poids du produit (Ex. 100 g, 1L, etc.)</li>
+</ul>
+<p>En dessous on trouve :</p>
+<ol>
+<li>Un champs de recherche, permettant de préciser sur quelle marque, un label, une catégorie on souhaite travailler,</li>
```
Comment (teolemon) [inline lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html:32]:
```suggestion
<li>Un champ de recherche, permettant de préciser sur quelle marque, un label, une catégorie on souhaite travailler,</li>
```

File: lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<p>Nous remercions les nombreux bénévoles qui nous ont déjà <a href="https://fr.openfoodfacts.org/aider-a-completer-les-produits">aidé à compléter des produits</a> et vous invitons à découvrir ce jeu. </p>
+<p><strong>Ce jeu permet de modifier des fiches produits simplement, rapidement et de façon ludique</strong>. De quoi rendre addictive la contribution à un projet Open Source.</p>
+<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">Les informations que vous ajoutez serviront à des millions de personnes pour choisir de meilleurs aliments et à des équipes de recherche en santé publique pour leurs études.</p>
+ <p class="callout panel">Pour jouer c'est simple, il vous suffit de vous connecter sur Open Food Fact puis de cliquer sur <a href="https://hunger.openfoodfacts.org/">ce lien</a>.</p>
+<p><strong>Envie d'en savoir plus avant de vous lancer ?</strong> Découvrez ci-dessous de plus amples informations sur le jeu !</p>
+	<h3>Comment fonctionnement cet Open DATA Game ?</h3>
```
Comment (teolemon) [inline lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html:14]:
```suggestion
	<h3>Comment fonctionne ce jeu ?</h3>
```

File: lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+<h2 class="subheader">La contribution à Open Food Facts devient un jeu d'enfant grâce à ce jeu imaginé et créé par les développeurs bénévoles d'Open Food Facts.
+</h2>
+
+<div class="button_div unselectbuttondiv_front_fr"><a href="https://hunger.openfoodfacts.org/" class="button">Jouer dès maintenant</button></a><div>
+<p>Nous remercions les nombreux bénévoles qui nous ont déjà <a href="https://fr.openfoodfacts.org/aider-a-completer-les-produits">aidé à compléter des produits</a> et vous invitons à découvrir ce jeu. </p>
+<p><strong>Ce jeu permet de modifier des fiches produits simplement, rapidement et de façon ludique</strong>. De quoi rendre addictive la contribution à un projet Open Source.</p>
+<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">Les informations que vous ajoutez serviront à des millions de personnes pour choisir de meilleurs aliments et à des équipes de recherche en santé publique pour leurs études.</p>
+ <p class="callout panel">Pour jouer c'est simple, il vous suffit de vous connecter sur Open Food Fact puis de cliquer sur <a href="https://hunger.openfoodfacts.org/">ce lien</a>.</p>
```
Comment (teolemon) [inline lang/fr/texts/contribuer-a-open-food-facts-en-jouant.html:12]:
```suggestion
 <p class="callout panel">Pour jouer c'est simple, il vous suffit de vous connecter sur Open Food Facts puis de cliquer sur <a href="https://hunger.openfoodfacts.org/">ce lien</a>.</p>
```

**Comment by stephanegigandet** [comment]
@teolemon : ok to merge? we can always refine later.

**Pull request merged**